Formula/gitversion.rb

```diff
@@ -1,21 +1,26 @@
 class Gitversion < Formula
   desc "Easy semantic versioning for projects using Git"
-  homepage "https://github.com/GitTools/GitVersion"
-  url "https://github.com/GitTools/GitVersion/releases/download/5.0.1/GitVersion-bin-fullfx-v5.0.1.zip"
-  sha256 "9b543d3e42e0d5e6fab0b44553cb6bbbb0e31431030ef761fc1a50c845fd166a"
+  homepage "https://gitversion.net"
+  url "https://github.com/GitTools/GitVersion.git",
+      tag:      "5.3.7",
+      revision: "b22e8726683a0997484b53c5efd54394e93b5306"
```

Comment:
Does this use submodules? Any other reason to prefer a checkout over a tarball e.g. https://github.com/GitTools/GitVersion/archive/5.3.7.tar.gz

Reply:
No submodules in use. I'll revert to tarball.